extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -27,14 +27,14 @@ class AmpStickyAd extends AMP.BaseElement {
   constructor(element) {
     super(element);
 
+    /** @const @private {!../../../src/service/vsync-impl.Vsync} */
+    this.vsync_ = this.getVsync();
```

Comment:
i'd like some clarification on what services are safe to retrieve in the constructor and the ones that need to be in buildCallback